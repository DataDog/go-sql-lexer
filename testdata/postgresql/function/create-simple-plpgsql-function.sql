CREATE OR REPLACE FUNCTION get_user_count() RETURNS integer AS $func$
BEGIN
  RETURN (SELECT COUNT(*) FROM users);
END;
$func$ LANGUAGE plpgsql;
