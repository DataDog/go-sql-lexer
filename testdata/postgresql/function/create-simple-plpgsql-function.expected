[
    {
        "expected": "CREATE OR REPLACE FUNCTION get_user_count ( ) RETURNS integer AS $func$BEGIN RETURN ( SELECT COUNT ( * ) FROM users ); END$func$ LANGUAGE plpgsql"
    },
    {
        "obfuscator_config": {
            "dollar_quoted_func": false
        },
        "expected": "CREATE OR REPLACE FUNCTION get_user_count ( ) RETURNS integer AS ? LANGUAGE plpgsql"
    }
]